boxoffice/models/line_item.py

```diff
@@ -1,16 +1,18 @@
 import itertools
+from sqlalchemy.sql import select, func
 from decimal import Decimal
 from collections import namedtuple
-from boxoffice.models import db, BaseMixin, Order, Item, DiscountPolicy
+from boxoffice.models import db, BaseMixin, Order, Item, DiscountPolicy, DiscountCoupon, DISCOUNT_TYPE
 from coaster.utils import LabeledEnum
 from baseframe import __
 
 __all__ = ['LineItem']
 
 
 class LINE_ITEM_STATUS(LabeledEnum):
-    CONFIRMED = (0, __("Confirmed"))
-    CANCELLED = (1, __("Cancelled"))
+    ADDED = (0, __("Added"))
+    CONFIRMED = (1, __("Confirmed"))
+    CANCELLED = (2, __("Cancelled"))
```

Comment:
What happens to all the existing rows in the database? These numbers can't change.